indico/web/assets/vars_js.py

```diff
@@ -117,6 +117,10 @@ def generate_global_file(config):
             },
 
             'Timetable': {
+                'entry': {
+                    'move': url_rule_to_js('timetable.move_timetable_entry'),
+                    'changeDates': url_rule_to_js('timetable.change_start_date')
```

Comment:
We are not changing date**s** here, but start `datetime`. What about `changeDatetime`?